compiler/crates/graphql-text-printer/tests/compact/fixtures/compact_test.expected

```diff
@@ -0,0 +1,19 @@
+==================================== INPUT ====================================
+query Test { 
+    #comments go away
+    aliasOfFoo : username(name : "val1"  ) @customDirective(level: 2) { #   and this comment as well
+        emailAddresses
+        firstName(if: false, unless: false)
+    }
+    ...FX
+}
+
+fragment FX on Query {
+    aliased : username(name : "argVal") {
+        thin:  emailAddresses
+    }
+}
+==================================== OUTPUT ===================================
+query Test{aliasOfFoo:username(name:"val1")@customDirective(level:2){emailAddresses,firstName(if:false,unless:false)},...FX}
+
+fragment FX on Query{aliased:username(name:"argVal"){thin:emailAddresses}}
```

Comment:
This test captures the same behaviour as GraphQLJava https://github.com/graphql-java/graphql-java/pull/2872